Updated, moved and Added Code and settings

TamaniWolf opened the pull request:
I updated and moved some parts of the code.
Added twitch oauth2 token creater/updater.
 '' Commands Ping/Reload.
 Added settings to the config.json.
 
I hope to give you same ideas.

TamaniWolf committed:
Updated index.js , exported wordreactions/eliza

Exported Word reactions to react/words.js
Changed 'const config' to 'const configmain'
Added Collections for commands, word reactions and twitch ouath2 token

TamaniWolf committed:
Updated config import

Updated config import
Changed twitch_client_id location to config/config.json

TamaniWolf committed:
Update package.json

Updated version of: axios, discord.js, humanize-duration, moment
Added: mathjs

TamaniWolf committed:
Created Twitch oauth2 token re-newer

TamaniWolf committed:
Moved twitch-monitor.js from ./ to ./twitch

TamaniWolf committed:
Moved minidb.js from ./ to ./twitch

TamaniWolf committed:
Moved live-embed.js from ./ to ./twitch

TamaniWolf committed:
Moved fooduse-monitor.js from ./ to ./twitch

TamaniWolf committed:
Moved eliza.js from ./ to ./twitch

TamaniWolf committed:
Moved discord-channel-sync.js from ./ to ./twitch

TamaniWolf committed:
Create data.json

TamaniWolf committed:
Exported word.js data from index.js

TamaniWolf committed:
Created command_handler for commands

TamaniWolf committed:
Created event_handler.js

Created event_handler.js for Discord Server and Clients events

TamaniWolf committed:
Create message.js

TamaniWolf committed:
Create ready.js

TamaniWolf committed:
Moved and Updated Config

Moved config.js from ./ to ./config
Updated Config with new settings

TamaniWolf committed:
Created roles.json

Created roles.json

TamaniWolf committed:
Create ping.js

TamaniWolf committed:
Create reload.js

TamaniWolf committed:
Create placeholder.js

Only a Placeholder

TamaniWolf committed:
Moved to ./twitch

TamaniWolf committed:
Moved to ./twitch

TamaniWolf committed:
Moved to ./twitch

TamaniWolf committed:
Moved to ./twitch

TamaniWolf committed:
Moved to ./twitch

TamaniWolf committed:
Moved to ./twitch

TamaniWolf committed:
Moved to ./config

TamaniWolf committed:
Update README.md

TamaniWolf committed:
Update config.js

TamaniWolf committed:
Update and rename data.json to .gitignore

TamaniWolf committed:
Moved data to ./twitch

TamaniWolf committed:
Update .gitignore

TamaniWolf committed:
Rename config.js to config-sample.js

TamaniWolf committed:
Update and rename roles.js to roles-sample.js

TamaniWolf committed:
Update LICENSE

GoudronViande24 commented:
If this bot now does create and update automatically the twitch oauth code, it would be sooo much better because you know it is not gonna stop working for no reasons.

TamaniWolf commented:
> 
> 
> If this bot now does create and update automatically the twitch oauth code, it would be sooo much better because you know it is not gonna stop working for no reasons.

Sadly i have to rework this, for  some reason it stopped updating ;;;;

TamaniWolf committed:
Delete yarn.lock

TamaniWolf committed:
Update package.json

TamaniWolf committed:
Update package.json

TamaniWolf committed:
Deleted twitch-api.js (Moved)

Deleted twitch-api.js (Moved to ./TimCore/Twitch )

TamaniWolf committed:
Deleted twitchrequest.js (Moved)

Deleted twitchrequest.js (Moved to ./TimCore/Twitch/twitchRequest )

TamaniWolf committed:
Deleted twitch-monitor.js (Moved)

Deleted twitch-monitor.js (Moved to ./TimCore/Twitch )

TamaniWolf committed:
Deleted minidb.js (Moved)

Deleted minidb.js (Moved to ./TimCore/Twitch )

TamaniWolf committed:
Deleted live-embed.js (Moved)

Deleted live-embed.js (Moved to ./TimCore/Twitch )

TamaniWolf committed:
Deleted fooduse-monitor.js (Moved)

Deleted fooduse-monitor.js (Moved to ./TimCore/Twitch )

TamaniWolf committed:
Delete eliza.js

TamaniWolf committed:
Deleted discord-channel-sync.js (Moved)

Deleted discord-channel-sync.js (Moved to ./TimCore/Twitch )

TamaniWolf committed:
Delete .gitignore

TamaniWolf committed:
Delete words.js

TamaniWolf committed:
Delete event_handler.js

TamaniWolf committed:
Deleted command_handler.js (Moved)

Deleted command_handler.js (Moved to ./TimCore/Commen/handlers )

TamaniWolf committed:
Delete message.js

TamaniWolf committed:
Delete ready.js

TamaniWolf committed:
Delete roles-sample.js

TamaniWolf committed:
Delete config-sample.js

TamaniWolf committed:
Delete placeholder.js

TamaniWolf committed:
Delete reload.js

TamaniWolf committed:
Delete ping.js

TamaniWolf committed:
Updated package's dependencies

Added:
-better-sqlite3
-chalk
-dotenv
-luxon
Updated:
-axios
-discordjs
-mathjs
Removed:
-monent

TamaniWolf committed:
Update index.js

Updated Code to work with Discord.js v13 and new/other dependencies
Moved parts of the code to other files.

TamaniWolf committed:
Create .env

TamaniWolf committed:
Create channels.js

TamaniWolf committed:
Create config.js

TamaniWolf committed:
Create lang.js

TamaniWolf committed:
Create roles.js

TamaniWolf committed:
Create twitch.js

TamaniWolf committed:
Create ping.js

TamaniWolf committed:
Create eliza.js

TamaniWolf committed:
Create words.js

TamaniWolf committed:
Create config.json

TamaniWolf committed:
Create ready.js

TamaniWolf committed:
Create messageCreate.js

TamaniWolf committed:
Create admin_command_handler.js

TamaniWolf committed:
Create event_handler.js

TamaniWolf committed:
Create twitch.js

TamaniWolf committed:
Create twitch-monitor.js

TamaniWolf committed:
Create twitch-api.js

TamaniWolf committed:
Create minidb.js

TamaniWolf committed:
Create live-embed.js

TamaniWolf committed:
Create fooduse-monitor.js

TamaniWolf committed:
Create discord-channel-sync.js

TamaniWolf committed:
Create twitchrequest.js

TamaniWolf committed:
Create twitchrequest.json

TamaniWolf committed:
Create .gitignore

TamaniWolf committed:
Create en_US.json

TamaniWolf committed:
Create twitch.json

TamaniWolf committed:
Add files via upload

TamaniWolf committed:
Delete twitch.json

TamaniWolf committed:
Create cc.json

TamaniWolf committed:
Add files via upload

TamaniWolf committed:
Delete cc.json

TamaniWolf committed:
Update package.json

added Original author
Updated version number

TamaniWolf committed:
Update README.md

TamaniWolf committed:
Update README.md

SugarD-x commented:
You have a small typo in one of your folder paths that is being referenced in several places. `Commen` should be `Common` in `./TimCore`.
